Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM decred/decred-golang-builder-1.9 as builder
+COPY . /go/src/github.com/decred/dcrd
+WORKDIR /go/src/github.com/decred/dcrd
+RUN dep ensure
+ENV CGO_ENABLED=0
+RUN mkdir -p /tmp/output
+RUN go build -ldflags "-s -w" -a -tags netgo -o /tmp/output/dcrd .
+WORKDIR /go/src/github.com/decred/dcrd/cmd
+RUN for cmd in `echo *`; do go build -ldflags "-s -w" -a -tags netgo -o /tmp/output/$cmd ./$cmd; done
+
+FROM alpine:latest
+RUN apk --no-cache add ca-certificates
+COPY --from=builder /tmp/output/* /usr/local/bin/
+VOLUME /var/lib/dcrd
+VOLUME /etc/dcrd
+EXPOSE 9108
+EXPOSE 9109
```

Comment:
Should add the testnet 19108 19109 ports as well, as I could have testnet=1 in my dcrd.conf.

Reply:
https://github.com/decred/dcrd/pull/919/commits/6368bfab2a67b0797c7133edc444433719951ce8